sections/video.liquid

```diff
@@ -31,64 +54,51 @@
     <noscript>
       <div
         class="video-section__media"
-        {% if section.settings.cover_image != blank %}
-          style="padding-bottom: {{ 1 | divided_by: section.settings.cover_image.aspect_ratio | times: 100 }}%;"
+        {% if poster != null %}
+          style="--ratio-percent: {{ 1 | divided_by: poster.aspect_ratio | times: 100 }}%;"
         {% endif %}
       >
-        <a
-          href="{{ section.settings.video_url }}"
-          class="video-section__poster media media--transparent media--landscape{% if section.settings.cover_image == blank %} video-section__placeholder{% endif %}"
-        >
-          {%- if section.settings.cover_image != blank -%}
-            {%- capture sizes -%}
-              {% if section.settings.full_width -%}
-                100vw
-              {%- else -%}
-                (min-width: {{ settings.page_width }}px) {{ settings.page_width | minus: 100 }}px, (min-width: 750px)
-                calc(100vw - 10rem), 100vw
-              {%- endif %}
-            {%- endcapture -%}
-            {%- assign alt = 'sections.video.load_video' | t: description: section.settings.description | escape -%}
-            {{
-              section.settings.cover_image
-              | image_url: width: 3840
-              | image_tag:
-                loading: 'lazy',
-                sizes: sizes,
-                widths: '375, 750, 1100, 1500, 1780, 2000, 3000, 3840',
-                alt: alt
-            }}
-          {%- else -%}
-            {{ 'collection-2' | placeholder_svg_tag: 'placeholder-svg placeholder' }}
-          {%- endif -%}
-        </a>
+        {%- if section.settings.video == null and section.settings.video_url != null -%}
+          <a
+            href="{{ section.settings.video_url }}"
+            class="video-section__poster media media--transparent{% if poster == null %} video-section__placeholder{% endif %}"
+          >
+            {%- if poster != null -%}
+              {{ poster
+                | image_url: width: 3840
+                | image_tag: loading: 'lazy', sizes: sizes, widths: '375, 750, 1100, 1500, 1780, 2000, 3000, 3840', alt: alt
+              }}
+            {%- else -%}
+              {{ 'collection-2' | placeholder_svg_tag: 'placeholder-svg placeholder' }}
+            {%- endif -%}
+          </a>
+        {%- else -%}
+          {{ section.settings.video
+            | video_tag:
+              image_size: "1100x",
+              autoplay: true,
```

Comment:
The video does autoplay though. It doesn't seem like it happen all the time but it will. I wonder if we should default not to autoplay here. 
I know we don't want to overthink the no js behaviour, but I think it's a safer bet to not autoplay since it's also not muted. 

[Video](https://screenshot.click/17-51-r3vj2-t0qya.mp4)

Reply:
+1

Reply:
Oh for just the no-js, yeah you're right, good call. Removed.